### PR TITLE
aarch64: Implicit selection of SLES

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by DrMullings** [comment]
LGTM

File: tests/installation/welcome.pm [hunk 1]
```diff
@@ -77,7 +77,9 @@ sub run {
     # On sle 15 license is on different screen, here select the product
     if (sle_version_at_least('15') && check_var('DISTRI', 'sle')) {
         # On s390x there will be only one product which means there is no product selection
-        return if check_var('ARCH', 's390x');
+        # On aarch64 we also have only one product at the moment because HPC was disabled temporarily
+        # (will change and not considered as bug)
+        return if (check_var('ARCH', 's390x') or check_var('ARCH', 'aarch64'));
```
Comment (okurz) [inline tests/installation/welcome.pm]: well, does not look good to me. The PRD clearly states that there is more than one product on aarch64 so this must not be accepted without recording a soft-fail. Please reconsider.

Comment (okurz) [reply tests/installation/welcome.pm]: @rwx788 can you make sure this is not overlooked?

Comment (rwx788) [reply tests/installation/welcome.pm]: It's in my list. I've also created progress ticket: https://progress.opensuse.org/issues/25922

**Pull request merged**